### PR TITLE
Add StateToBoolConverter

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] `StateToBoolConverter ` <https://github.com/CommunityToolkit/Maui/issues/74>

### CODE REVIEW

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/state-to-bool-converter.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+> [!NOTE]
+> Note that the expected `LayoutState` can be supplied in the following order of precedence:
+> 1. as the `ConverterParameter` in the converter binding; this supersedes the `StateToCompare` property
+> 2. as the `StateToCompare` property on the converter
+
+The `ConvertBack` method is not supported.
+
+## Syntax
+
+The following example shows how to use the converter to change the visibility of a `Label` control based on the `LayoutState` property which is modified on a `Button` `Command`.ß
```
Comment (jfversluis) [inline docs/maui/converters/state-to-bool-converter.md:33]:
```suggestion
The following example shows how to use the converter to change the visibility of a `Label` control based on the `LayoutState` property which is modified on a `Button` `Command`.
```

File: docs/maui/converters/state-to-bool-converter.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.StateToBooleanConverterPage">
+
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <mct:StateToBooleanConverter x:Key="StateToBooleanConverter" StateToCompare="Success" />
```
Comment (jfversluis) [inline docs/maui/converters/state-to-bool-converter.md:47]:
```suggestion
            <toolkit:StateToBooleanConverter x:Key="StateToBooleanConverter" StateToCompare="Success" />
```

**Review by jfversluis — CHANGES_REQUESTED** [review]
One weird German character and I think you mismatched the xmlns?

Not 100% sure so I'll leave it to you to accept the suggestions or fix it otherwise

File: docs/maui/converters/state-to-bool-converter.md (hunk 1 above)
Comment (bijington) [reply docs/maui/converters/state-to-bool-converter.md:33]: damn that was my attempt at pressing CMD + S to save the doc but hitting ALT instead. Good catch!

**Comment by PRMerger19** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger19** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger13** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by jfversluis — APPROVED** [review]

**Pull request merged**